src/main/java/uk/gov/hmcts/darts/annotation/service/impl/AnnotationDataManagement.java

```diff
@@ -62,7 +62,7 @@ public InputStreamResource download(ExternalObjectDirectoryEntity externalObject
             return new InputStreamResource(downloadResponseMetaData.getInputStream());
         } catch (IOException e) {
             log.error("Failed to download annotation document {}",
```

Comment:
Please can you check this error message - we don't know what EOD was used.
So I think it makes sense for it just to be the `annotation_document` ID here instead?

Reply:
changed to annotation_doc ID